Regressions In-Depth.Rmd

```diff
@@ -0,0 +1,72 @@
+---
+title: "Regressions In-Depth"
+output: html_document
```

Comment:
This is an "R Markdown" document (.Rmd) that includes both code and output. To see what the result is, open the .html file I uploaded.